sql/monitoring/telemetry_missing_columns_v2/view.sql

```diff
@@ -17,7 +17,7 @@ extracted AS (
     `moz-fx-data-shared-prod`.udf.extract_document_version(_TABLE_SUFFIX) AS document_version,
     additional_properties
   FROM
-    `moz-fx-data-shared-prod.telemetry_live.*`
+    `moz-fx-data-shared-prod.telemetry_stable.*`
```

Comment:
@acmiyaguchi From my reading of this query, we don't ever read data from the current day. Is there any reason not to hit stable rather than live tables?

Reply:
This was because this query was developed in `moz-fx-data-shar-nonprod-efed`, where the stable tables do not exist. It should be fine to use the stable table and use the most recent date.